ngraph/python/tests/test_onnx/utils/onnx_backend.py

```diff
@@ -0,0 +1,131 @@
+# ******************************************************************************
```

Comment:
It would be nice if OV could have a proper onnx backend to use by customers.

Reply:
Agreed. Not available at this point, unfortunately. We can move it from tests to official API later.